moto/sqs/models.py

```diff
@@ -834,9 +841,11 @@ def send_message_batch(self, queue_name, entries):
             raise TooManyEntriesInBatchRequest(len(entries))
 
         messages = []
+
+        failedInvalidDelay = []
         for entry in entries.values():
             # Loop through looking for messages
-            message = self.send_message(
+            message, invalidDelay = self.send_message(
```

Comment:
The `DelaySeconds`-parameter can also part of the `boto3.client("sqs").send_message(..)`-call, so IMO we should stick with the initial implementation. Raising the error in `def send_message()` ensures that the validation happens in both scenarios, when calling `send_message` and `send_message_batch`.

That just means that `send_message_batch` should catch the error, something like:
```
try:
    message = self.send_message(..)
    messages.append(message)
except ..:
    failedInvalidDelay.append(entry)
```

Reply:
Ok sir this part I understood and implemented it.